### PR TITLE
Data repo - Fake prod project pool

### PR DESCRIPTION
**Pull request by snf2ye**
Data repo has a legacy instance that we refer to as "fake prod." We hope to retire it soon, but in the mean time, it's still in use. It lives in and references a different google project folder than our "real production" instance, so we need a project pool that can live in this different folder.

### CODE REVIEW

**Review by nmalfroy — APPROVED** [review]
thanks!

**Comment by samanehsan** [comment]
We'll also need to update the `RBS_POOL_ID ` in the fake prod config: https://github.com/broadinstitute/datarepo-helm-definitions/blob/master/fakeprod/datarepo-api.yaml#L24

File: src/main/resources/config/prod/resource-config/datarepo_fakeprod_v1.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Data Repo buffered workspace template
+---
+configName: "datarepo_fakeprod_v1"
+gcpProjectConfig:
+  projectIdSchema:
+    prefix: "datarepo"
+    scheme: "RANDOM_CHAR"
+  parentFolderId: "270278425081" #data.test-terra.bio/repos/jade-dev
```
Comment (yonghaoy) [inline src/main/resources/config/prod/resource-config/datarepo_fakeprod_v1.yml:8]: have you granted RBS SA permission to that folder?

**Review by yonghaoy — COMMENTED** [review]
It is a 'fake' prod instance, is that used in production? If not, I think we should not use prod RBS, but use dev/tools instead.

File: src/main/resources/config/prod/resource-config/datarepo_fakeprod_v1.yml (hunk 1 above)
Comment (snf2ye) [reply src/main/resources/config/prod/resource-config/datarepo_fakeprod_v1.yml]: Yes, the folder is shared between our dev, alpha, staging, etc. environments, so it should already RBS SA permission.

Comment (yonghaoy) [reply src/main/resources/config/prod/resource-config/datarepo_fakeprod_v1.yml]: Yup, dev RBS does have access on that folder.  If using dev RBS, no terraform change is needed!

File: src/main/resources/config/dev/resource-config/datarepo_fakeprod_v1.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Data Repo buffered workspace template
+---
+configName: "datarepo_fakeprod_v1"
+gcpProjectConfig:
+  projectIdSchema:
+    prefix: "datarepo"
```
Comment (yonghaoy) [inline src/main/resources/config/dev/resource-config/datarepo_fakeprod_v1.yml:6]: nit: not sure if you want to use `datarepo-dev` as prefix to differentiate with `prod` projects

File: src/main/resources/config/dev/resource-config/datarepo_fakeprod_v1.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    scheme: "RANDOM_CHAR"
+  parentFolderId: "270278425081" #data.test-terra.bio/repos/jade-dev
+  billingAccount: "0106B0-41CAA9-427C96"
```
Comment (yonghaoy) [inline src/main/resources/config/dev/resource-config/datarepo_fakeprod_v1.yml:9]: Is this prod billing accounts? I saw other dev data repo is using 01A82E-CA8A14-367457

**Review by yonghaoy — APPROVED** [review]

**Review by samanehsan — APPROVED** [review]

**Pull request merged**